### DISCUSSION TITLE
是否有vue 2.x版本的啊

### DISCUSSION THREAD

**Discussion by JinPengGeng** [vbenjs/vue-vben-admin#339] [Q&A]
1、是否有vue 2.x版本的啊？我们现在需要兼容IE，上不了 vue 3.X
2、如果现在使用 vue 2.x + composition-api 是否可行？
希望大神给个意见

**Comment by anncwb** [comment]
有，不过没维护被我关闭了。composition-api 可以直接用，没大问题。我自己也在用

**Reply by yuguaa** [reply]
> 有，不过没维护被我关闭了。composition-api 可以直接用，没大问题。我自己也在用

能开放一下吗？这个项目封装的有点多，感觉看起来太吃力，想从vue2.x熟悉的啃一啃，看懂整体的思考先

**Reply by IFnGSiYu** [reply]
作为一个前端菜鸟，看起来确实吃力，封装的有点多，尤其是类型声明这块😢

**Reply by repairearth** [reply]
+1，能否开放 v2 的仓库看下 🤝

**Reply by repairearth** [reply]
如果直接在这个项目，把 vue3 替换成 vue2 + composition-api 可以跑吗

**Reply by 58702064** [reply]
初学者要学习面向对象的开发方式。高复用。

**Comment by snowchenlei** [comment]
2.x为啥不用ele-admin
